src/main/java/com/sfl/cronofy/api/model/request/FreeBusyRequest.java

```diff
@@ -144,8 +144,12 @@ public void setIncludeManaged(final Boolean includeManaged) {
         this.includeManaged = includeManaged;
     }
 
-    public List<String> getCalendarIds() {
-        return calendarIds;
+    public Object[] getCalendarIds() {
```

Comment:
Mostly because my Java is rusty so I naively used `Object[]` as the argument it is passed to is `Object...`. I'll sort it out.